src/dynatrace/azext_dynatrace/aaz/latest/dynatrace/monitor/_create.py

```diff
@@ -0,0 +1,658 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "dynatrace monitor create",
+)
+class Create(AAZCommand):
+    """Create a MonitorResource
+    """
+
+    _aaz_info = {
+        "version": "2021-09-01-preview",
```

Comment:
Seems we should use stable version https://github.com/Azure/azure-rest-api-specs/tree/main/specification/dynatrace/resource-manager/Dynatrace.Observability/stable/2021-09-01

Reply:
The service return command does not support the 2021-09-01 version

Reply:
@divyansh3131, could you please help confirm if 2021-09-01 version (stable version) is available for service?

Reply:
Service team mentioned the service supports 2021-09-01 version now, could you test it again? @PARADISSEEKR